lib/trento/users.ex

```diff
@@ -18,7 +18,7 @@ defmodule Trento.Users do
   @spec by_id(id :: non_neg_integer()) :: {:ok, User.t()} | {:error, :not_found}
   def by_id(id) do
     case Repo.get(User, id) do
-      %User{} = user -> {:ok, user}
+      %User{} = user -> {:ok, User.polish_username(user)}
```

Comment:
question: are we sure of this? This `by_id` is used by Pow in many places and it could have some side-effect.
I find it safer to use `polish_name` in the `get_activity_actor` function.

Reply:
whoops, let me fix that 🙈

Reply:
You are right. I confused the function with `get_by`. 
We have `by_id`, `get_by` (used by `Pow`), `get_user`...
I don't know why we have `by_id` and `get_user`... but well.
Yes, i would prefer a more informative name maybe. I wouldn't have specific things in general naming functions